client/app/scripts/superdesk-packaging/views/add-to-package.html

```diff
@@ -0,0 +1 @@
+<div sd-add-package-dropdown></div>
```

Comment:
Ah thanks, didn't know about that. Leave it then as is, of course, if that is the agreed approach.